system.md

```diff
@@ -6,4 +6,87 @@ order: 4
 menu: contest
 ---
 
-To be announced.
\ No newline at end of file
+The system is based on Ubuntu 18.04 LTS.
+It will be possible to submit solutions using C99, C++17, Java 11, Python 2.7, and Python 3.6.
+Kotlin will also be available for company teams.
+
+## Documentation
+Documentation for C, C++, Java, Python 2, Python 3, and Kotlin will be available through [zeal](https://zealdocs.org/).
+
+## Compilation options
+(NOTE: these are subject to minor changes before the contest)
```

Comment:
Double line breaks for a new paragraph, now these parenthesis are ugly
And "Note" bold instead of caps?

Reply:
More line breaks everywhere may make sense